vss-processor/build.gradle.kts

```diff
@@ -52,19 +58,57 @@ tasks.withType<Test>().configureEach {
     }
 }
 
-configure<Publish_gradle.PublishPluginExtension> {
+publish {
     mavenPublicationName = "release"
     componentName = "java"
     description = "Vehicle Signal Specification (VSS) Code Generator for the KUKSA SDK"
 }
 
 tasks.register("javadocJar", Jar::class) {
     dependsOn("dokkaHtml")
+
+    val buildDir = layout.buildDirectory.get()
     from("$buildDir/dokka/html")
     archiveClassifier.set("javadoc")
 }
 
+tasks.withType<DokkaTask>().configureEach {
+    notCompatibleWithConfigurationCache("https://github.com/Kotlin/dokka/issues/2231")
+}
+
 java {
     withJavadocJar() // needs to be called after tasks.register("javadocJar")
     withSourcesJar()
 }
+
+// Tasks for included composite builds need to be called separately. For convenience sake we depend on the most used
+// tasks. Every task execution of this project will then be forwarded to the included build project. Since this module
+// is hard coupled to the
+//
+// We have to manually define the task names because the task() method of the included build throws an error for any
+// unknown task.
+val dependentCompositeTasks = setOf(
+    "publishToMavenLocal",
+    "publishAllPublicationsToOSSRHReleaseRepository",
+    "test",
```

Comment:
test is a really useful addition but it seems like it is not working?

neither `./gradlew test`, `./gradlew build` nor `./gradlew vss-processor:test` leads to the execution of the vss-processor-plugin tests, can you check this?